lib/encode.js

```diff
@@ -0,0 +1,122 @@
+const { typeOf, humanize } = require('./utils');
+const INPUT_TABLE = require('./inputs');
+
+const SHORTHAND_FLAGS = {
+  '*': { _required: true },
+  '!': { _disabled: true },
+  '@': (model) => ({
+    _type: 'association',
+    _model: model
+  })
+  // '*': 'required',
+  // '@': 'model',
+  // ':': 'type',
+  // '!': 'pattern',
+  // '!': 'disabled'
+  // ':': '_input',
+  // '/': '_disabled'
```

Comment:
You can kill this line

Reply:
```
  // ':': 'type',
  // ':': '_input',
  // '?': '_help'
  // '()': '_placeholder'
  // '_': '_placeholder'
```

After all the updates, that's what it looks like ^